third_party/move/move-prover/lab/data/new-boogie-aptos-stdlib/mod_by_mod_boogie_3.2.4.svg

```diff
@@ -0,0 +1,918 @@
+<svg width="1000" height="3630" viewBox="0 0 1000 3630" xmlns="http://www.w3.org/2000/svg">
```

Comment:
However, in this graphic it looks the new boogie is much better?

Reply:
Yeah, to compare the performance, between 3.0.9 and 3.2.4, we should read the graph because the data is collected on the same machine.